pkg/apis/core/v1alpha1/defaults.go

```diff
@@ -321,6 +321,34 @@ func SetDefaults_Worker(obj *Worker) {
 	}
 }
 
+// SetDefaults_ClusterAutoscaler sets default values for ClusterAutoscaler object.
+func SetDefaults_ClusterAutoscaler(obj *ClusterAutoscaler) {
+	if obj.ScaleDownDelayAfterAdd == nil {
+		obj.ScaleDownDelayAfterAdd = &metav1.Duration{Duration: 1 * time.Hour}
+	}
+	if obj.ScaleDownDelayAfterDelete == nil {
+		obj.ScaleDownDelayAfterDelete = &metav1.Duration{Duration: 0}
```

Comment:
In the documentation you have 
> ... defaults to scanInterval (defaults to ScanInterval).

Reply:
This is more related to CA's flag behavior where setting it to `0` defaults it to ScanIntervel. Refer [here](https://github.com/gardener/autoscaler/blob/machine-controller-manager-provider/cluster-autoscaler/main.go#L93-L94)

Reply:
But this is something internal to the cluster autoscaler and Gardener does not need to know about it. So, please set in the documentation that the default value is `0`.

Reply:
Okay understood. Apologies for the confusion. I have [fixed the docs](https://github.com/gardener/gardener/pull/4508/commits/eaed0c70e3822c2a43c0c497bbfd8ab5ea4af716) and combined all the recent suggestions in the same commit.